<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>250</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>96</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>96</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>96</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Tree mApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>250</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK KR</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="button_next">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>180</y>
        <width>85</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(211, 215, 207);
font: 13pt &quot;Noto Sans CJK KR&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>NEXT</string>
      </property>
     </widget>
     <widget class="QLabel" name="foldername">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>461</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Noto Sans CJK KR&quot;;</string>
      </property>
      <property name="text">
       <string>Choose a folder of images to create a mosaic</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="title_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>141</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Noto Sans CJK KR&quot;;</string>
      </property>
      <property name="text">
       <string>Pick a folder</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_browse">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>85</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);
border: none;
font: 12pt &quot;Noto Sans CJK KR&quot;;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/folder_icon.png</normaloff>:/icons/folder_icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="button_create">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>180</y>
        <width>85</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 96, 100);
font: 12pt &quot;Noto Sans CJK KR&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>CREATE</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_yes">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Noto Sans CJK KR&quot;;</string>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_no">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Noto Sans CJK KR&quot;;</string>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
     <widget class="QLabel" name="title_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>581</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Noto Sans CJK KR&quot;;</string>
      </property>
      <property name="text">
       <string>Do you want to copy the images to the computer?</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Progress thingy</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>We did it!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Do it again?</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>141</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string> I am Error</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
